### PR TITLE
[doc] Update routing and non routing index on MAPS.md

### PR DESCRIPTION
**Pull request by frodrigo**
I keep the Steps part of MAPS.md unchanged, as I not sure of current steps order (step 6).

### CODE REVIEW

File: docs/MAPS.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # MWM Files
 
 MAPS.ME uses maps in its own vector format, MWM. It contains classified features sorted and simplified by zoom level.
-For car routing, it needs a separate routing index in a `.mwm.routing` file. We build maps for the entire planet:
+We build maps for the entire planet:
```
Comment (Zverik) [inline docs/MAPS.md]: I think the removed part should be replaced with "It also can include a pre-calculated routing index for car routing."

**Comment by Zverik** [comment]
Step 6 should also be rewritten to "building routing indices out of `*.osrm*` files".

**Comment by frodrigo** [comment]
Requested changes done

**Comment by Zverik** [comment]
Thanks!

**Pull request merged**